### DISCUSSION TITLE
What is @next/react-dev-overlay exactly?

### DISCUSSION THREAD

**Discussion by corysimmons** [vercel/next.js#21601] [Help] [answered]
No usage docs or screenshots. Google just shows npmjs.com clones. Kinda curious what it is and how to use it.

**Comment by Lazzaro83** [comment, marked as answer]
https://stackoverflow.com/questions/70665580/what-is-the-purpose-of-next-react-dev-overlay/70681184#70681184

**Reply by corysimmons** [reply]
oooooooooooooh. Thank you. :)
